integration/tests/cook/test_basic.py

```diff
@@ -209,6 +209,29 @@ def test_mea_culpa_retries(self):
         finally:
             util.kill_jobs(self.cook_url, [uuid])
 
+    def test_compute_cluster(self):
+        settings_dict = util.settings(self.cook_url)
+        # For now, we only have one compute cluster. This could be wrong with future
+        # refactors, but for now, this is the only compute cluster we have, so we should
+        # expect it.
+        expected_compute_cluster = settings_dict['mesos-compute-cluster-name']
+        expected_mesos_framework = settings_dict['mesos-framework-id']
+        job_uuid, resp = util.submit_job(self.cook_url)
+
+        try:
+            self.assertIsNotNone(expected_compute_cluster)
```

Comment:
Instead of this assert, can we do a `skipUnless`? That way, the latest integration tests will still work until the change is released.

Reply:
Don't understand why. This integration test works as-is right now. That field is defaulted in config.clj, so it will always exist unless either a bug is introduced, or someone tries running these new tests on an old cook. One of which we want to fail, and the other is expected behavior.

Reply:
:+1: